src/main/java/frc/robot/Field.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+
+import edu.wpi.first.apriltag.AprilTag;
+import edu.wpi.first.apriltag.AprilTagFieldLayout;
+import edu.wpi.first.wpilibj.Filesystem;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class Field {
+
+    // Wrapper for loading an AprilTagFieldLayout from a JSON file
+    public static AprilTagFieldLayout getApriltagLayout() {
+        String absolutePath = Filesystem.getDeployDirectory().getAbsolutePath();
+
+        AprilTagFieldLayout layout = new AprilTagFieldLayout(new ArrayList<>(), 0, 0);
+        Path jsonPath = Path.of(absolutePath + "/apriltags.json");
+        try {
+            layout = new AprilTagFieldLayout(jsonPath);
+            String outp = "";
+            for (AprilTag tag : layout.getTags()) {
```

Comment:
You can use [String.join()](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#join-java.lang.CharSequence-java.lang.CharSequence...-) here